core/src/wallet/tezos/explorers/NodeTezosLikeBlockchainExplorer.cpp

```diff
@@ -44,6 +44,7 @@ namespace ledger {
                 TezosLikeBlockchainExplorer(configuration, {api::Configuration::BLOCKCHAIN_EXPLORER_API_ENDPOINT}) {
             _http = http;
             _parameters = parameters;
+            _explorerVersion = configuration->getString(api::Configuration::BLOCKCHAIN_EXPLORER_VERSION).value_or("v3");
```

Comment:
I suggest to store somewhere - configuration file, `constexpr` variable - this hardcoded value "v3"

Reply:
Nice catch, I already did something for that 🤔 I will use it there